src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp

```diff
@@ -3212,7 +3273,6 @@ size_t CorProfiler::CallTarget_RequestRejitForModule(ModuleID module_id, ModuleM
     if (!vtMethodDefs.empty())
     {
         this->rejit_handler->EnqueueForRejit(vtModules, vtMethodDefs);
-        this->rejit_handler->RequestRejitForNGenInliners();
```

Comment:
What was the issue with RequestRejitForNGenInliners being called here?

Reply:
Oh right, that modification belongs to the `remove the json` PR, and the reason for that is we need to call the RequestReJIT on proper threads or callbacks, for the ModuleFinished callback is ok to call it directly, but if we do a PInvoke (like in the remove the json PR) then a direct call to RequestReJIT will fail. I'll revert this change on this PR.

Reply:
done in 59ae73f